react-frontend/src/components/DigitalPrinciples/digitalPrinciples.js

```diff
@@ -1,26 +1,24 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import BannerSideImage from '../PageElements/BannerSideImage/bannerSideImage';
 
-import {
-  SimpleTextContainer,
-  GlobaStyleSize,
-} from '../StyleComponents/pageContent';
-import {
-  SimpleTextPageTitle,
-  DigitalPrinciplesHeading,
-} from '../StyleComponents/headings';
+import { PageContainer, GlobaStyleSize } from '../StyleComponents/pageContent';
+import { DigitalPrinciplesHeading } from '../StyleComponents/headings';
 import { HrefLink } from '../StyleComponents/htmlTags';
 
+const digitalPrinciplesImage = require('../../images/pngIllustrations/digitalPrinciplesWhite.png');
+
 function DigitalPrinciples() {
   return (
     <DocumentTitle title="Digital Principles - Digital Government - Province of British Columbia">
-      <SimpleTextContainer>
+      <PageContainer>
         <GlobaStyleSize />
-        <SimpleTextPageTitle>
-          Digital Principles for the Government of British Columbia (Version
-          1.0)
-        </SimpleTextPageTitle>
-
+        <BannerSideImage
+          title={
+            'Digital Principles - Digital Government - Province of British Columbia'
```

Comment:
Just noticed this - could you please change the title in the banner back to what it was: "Digital Principles for the Government of British Columbia (Version 1.0)?"

I know this is confusing, but this is one case where I think a more succinct title for the `<title>` works for its Accessibility function ... but the text in the `<h1>` needs to remain.

Thanks!